docs/sprint6.md

```diff
@@ -50,14 +50,12 @@ Objectives:
 
 1. Write client-side AJAX to delete items when the delete button is clicked.
 
-1. Make sure the deleted input is removed as well.
```

Comment:
I think this is repetitive with the item 2 down: Ensure that the song list on the page (the main album row that contains this song) is updated as well.